src/analytics/Properties.tsx

```diff
@@ -632,6 +632,8 @@ interface RequestEventsProperties {
   [RequestEvents.request_confirm_back]: undefined
   [RequestEvents.request_confirm_request]: {
     requesteeAddress: string
+    recipientType?: RecipientType
+    isScan: boolean
```

Comment:
nit: would be great to align the field names with the types (instead of `isScan` and `isFromScan` having 2 different names), and maybe to make them a bit more self-documenting too. I'd probly go with `isFromQRScan`

Reply:
this property is being added to an existing event, so there isn't a backwards compatibility issue to just changing it here. But keeping it consistent with the property names on other events sounds fine